zilencer/management/commands/sync_api_key.py

```diff
@@ -16,9 +16,12 @@ def handle(self, *args, **options):
         config_file = os.path.join(os.environ["HOME"], ".zuliprc")
         if not os.path.exists(config_file):
             raise RuntimeError("No ~/.zuliprc found")
-        config = SafeConfigParser()
+        config = ConfigParser()
         with open(config_file, 'r') as f:
-            config.readfp(f, config_file)
+            if hasattr(config, 'read_file'):
+                config.read_file(f, config_file)
+            else:
+                config.readfp(f, config_file)
```

Comment:
well that's lame.  Seems like a bug in `six`.  Can you add a comment explaining the issue, at least?